tools/deploy.py

```diff
@@ -53,17 +49,36 @@ def deploy_pew_pew_control_module(containerPath):
 def deploy_py3status_module():
     configHomePath = Path(os.getenv("XDG_CONFIG_HOME", "~")).expanduser()
     candidates = [
-        configHomePath / ".config/py3status/modules",
-        configHomePath / ".config/i3status/py3status",
-        configHomePath / ".config/i3/py3status",
-        configHomePath / ".i3/py3status",
+        configHomePath / ".config" / "py3status" / "modules",
+        configHomePath / ".config" / "i3status" / "py3status",
+        configHomePath / ".config" / "i3" / "py3status",
+        configHomePath / ".i3" / "py3status",
     ]
```

Comment:
Not really necessary to split these here, as I strongly doubt that we'll ever port this to Windows :D the other form is more compact and less visual noise.